web/src/buddy_data.ts

```diff
@@ -142,29 +142,33 @@ function get_num_unread(user_id: number): number {
 }
 
 export function user_last_seen_time_status(user_id: number): string {
+    // Check if the user is deactivated
+    if (people.get_non_active_human_ids().includes(user_id)) {
+        // Determine if user is a bot or human
+        const user = people.get_by_user_id(user_id);
+        if (user.is_bot) {
+            return $t({defaultMessage: "This bot has been deactivated."});
+        } else {
+            return $t({defaultMessage: "This user has been deactivated."});
+        }
```

Comment:
"This bot has been deactivated." and "This user has been deactivated." should be italicized as per requirement stated in the issue.

Reply:
The commit message for this commit is not line-wrapped properly.
Check out [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#formatting-guidelines) for more details.